src/peft/tuners/poly/router.py

```diff
@@ -37,12 +37,10 @@ def get_router(poly_config: PolyConfig) -> nn.Module:
 
 class Router(nn.Module, ABC):
     @abstractmethod
-    def reset(self):
-        ...
+    def reset(self): ...
```

Comment:
Let's undo all the changes in this file. Make sure that the `ruff --version` you use is `0.4.10`.